controllers/node.go

```diff
@@ -598,7 +652,12 @@ func deleteNode(w http.ResponseWriter, r *http.Request) {
 		returnErrorResponse(w, r, formatError(err, "internal"))
 		return
 	}
-
+	node.Action = models.NODE_DELETE
+	go func() {
+		if err := mq.NodeUpdate(&node); err != nil {
+			logger.Log(1, "error publishing node delete "+err.Error())
```

Comment:
no need to concatenate strings, just do "error publishing node delete", err.Error(). The logger will join with spaces.

Reply:
updated this and other logger.Log calls